cypress-io/cypress
# chore: add test replay button to debug page

### Additional details


### Steps to test
1. Create/link to repository with failing tests
2. Navigate to Debug page
3. Verify Presence of "View Replay" button on test failures

### How has the user experience changed?


### PR Tasks


- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
